### PR TITLE
Inactive player kick

### PR DESCRIPTION
**Pull request by Loki1976**
Kick idle players after (x) minutes.
Server property KICK_IDLE_PLAYER_STATUS - enable/disable default disabled.
Server property KICK_IDLE_PLAYER_TIME - time in mintures

### CODE REVIEW

**Comment by Graveen** [comment]
Nice ! Can you correct your PR for the appveyor error: 

Errors and Failures:
1) Test Failure : DOL.Server.Tests.FriendsManagerTest.FriendsManager_RemovePlayer_RetrieveEmptyFriendsList
     Expected: <empty>
  But was:  < "buddy" >
at NUnit.Framework.CollectionAssert.IsEmpty(IEnumerable collection)
at DOL.Server.Tests.FriendsManagerTest.FriendsManager_RemovePlayer_RetrieveEmptyFriendsList() in C:\projects\dolsharp\IntegrationTests\Managers\FriendsManagerTest.cs:line 80

Thank you VM. I think this feature is interesting !

File: GameServer/managers/playermanager/FriendsManager.cs [hunk 1]
```diff
@@ -123,21 +123,25 @@ public void RemovePlayerFriendsListFromCache(GamePlayer Player)
...
-			{
-				if (log.IsWarnEnabled)
-					log.WarnFormat("Gameplayer ({0}) was not registered in Friends Manager Cache while trying to remove!", Player);
-			}
-			
-			FriendStatus[] offlineFriends;
-			if (!PlayersFriendsStatusCache.TryRemove(Player, out offlineFriends))
+			string[] friends = null;
+			//<**loki**>
+			if (friends != null)
```
Comment (Graveen) [inline GameServer/managers/playermanager/FriendsManager.cs:128]: I think this is the issue with unit tests.

Comment (Loki1976) [reply GameServer/managers/playermanager/FriendsManager.cs:128]: I did this to avoid ugly console spam on kicking player only. Ill correct

**Comment by Graveen** [comment]
TY mate.

**Pull request merged**